vite.config.ts

```diff
@@ -70,6 +70,10 @@ export default () => {
       manifest: true,
       target: browserslistToEsbuild(),
       reportCompressedSize: false,
+      terserOptions: {
```

Comment:
@hanspagel the thing I'm contending with is in staging the `Collection` class is being changed to `R`, so the logic above where you removed the numbers isn't working, we need to prevent the classname mangling altogether.

Thought this would do it, but it seems not to make a difference